src/dma.rs

```diff
@@ -324,14 +331,14 @@ macro_rules! impl_channel {
                     handle:  &mut Handle,
                     address: u32,
                 ) {
-                    handle.dma.$chfield.par.write(|w| w.pa().bits(address));
+                    unsafe { handle.dma.$chfield.par.write(|w| w.pa().bits(address)); }
```

Comment:
This confuses me. Wouldn't the `bits` method be unsafe for all targets? Why did it work in the first place?